packages/dashmate/src/listr/tasks/setup/local/configureCoreTaskFactory.js

```
@@ -301,6 +347,47 @@
+
+                let blocksGenerated = 0;
+
+                const blocksToGenerateInOneStep = 10;
+
+                do {
+                  ({
+                    result: blockchainInfo,
+                  } = await ctx.seedCoreService.getRpcClient().getBlockchainInfo());
+
+                  isV20Activated = blockchainInfo.softforks && blockchainInfo.softforks.v20
+                    && blockchainInfo.softforks.v20.active;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)